src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/JsonArrayPool.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Buffers;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Api.Features.Formatters
+{
+    internal class JsonArrayPool : IArrayPool<char>
```

Comment:
Might be nice to explain why this class exists. Looks like a pass-through wrapper.

Reply:
Yep agree, I was still working through trying to get things running. The class we were using got marked as internal so its no longer available. This adapts the Json.net IArrayPool to the .NET ArrayPool